providers/NMSReflectionProvider/src/main/java/net/ess3/nms/refl/providers/ReflFormattedCommandAliasProvider.java

```diff
@@ -0,0 +1,60 @@
+package net.ess3.nms.refl.providers;
+
+import net.ess3.nms.refl.ReflUtil;
+import net.ess3.provider.FormattedCommandAliasProvider;
+import org.bukkit.command.CommandSender;
+import org.bukkit.command.FormattedCommandAlias;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+
+public class ReflFormattedCommandAliasProvider extends FormattedCommandAliasProvider {
+
+    private final boolean paper;
+    private final Field formatStringsField;
+    private final Method buildCommandMethod;
```

Comment:
`MethodHandle`s are more performant and should be used instead of java.lang.reflect's Method

See `ReflServerStateProvider` if you need to know how we use `MethodHandle`

Reply:
Sure thing!